Add support for AKS paid SKU

ThorstenHans opened the pull request:
This PR adds support for AKS paid SKU. 

The default value of `sku_tier` is set to `Free` to ensure backward compatibility

yupwei68 commented:
Hi @ThorstenHans , thanks for opening this PR. Overall it looks fine to me. Could you please 1. add this feature into fixture/test, and test if works when it's assigned "Paid"  2. add this feature into the example of REAME.

ThorstenHans commented:
@yupwei68 

I have added an AKS instance to the test fixture and verified creation by checking the resulting AKS id (ARM doesn't expose SKU tier on AKS resource).

Additionally, I have added the `sku_tier` to the sample shown in the README as suggested.

yupwei68 reviewed (approved):
LGTM!
